tests/lib/rules/no-private-routing-service.js

```diff
@@ -0,0 +1,54 @@
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require('../../../lib/rules/no-private-routing-service');
+const RuleTester = require('eslint').RuleTester;
+
+const { ERROR_MESSAGE } = rule;
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const ruleTester = new RuleTester({
+  parser: require.resolve('babel-eslint'),
+  parserOptions: {
+    ecmaVersion: 2015,
+    sourceType: 'module',
+  },
+});
+
+ruleTester.run('no-private-routing-service', rule, {
+  valid: [
+    "export default Component.extend({ routing: service('routing') });",
+    "export default Component.extend({ routing: service('-router') });",
```

Comment:
The service names in these valid test cases are confusingly similar to the actual service name. Can you switch these to using some obviously fake names like `someServiceName` or `otherServiceName` to make it clear these are testing what happens when a non-real name is used?

Reply:
That was by design, to ensure that it was the arg to `service` that was linted, not the Property's key

Reply:
Yeah, so I'm suggesting `service('someServiceName')`.

Reply:
ahh gotcha. Added something similar [here](https://github.com/ember-cli/eslint-plugin-ember/pull/694/files#diff-e8b0328364cdaae6b11a4613ade481a0R27)